.github/workflows/smoketest.yml

```diff
@@ -231,10 +231,18 @@ jobs:
         sudo apt-get update
         sudo apt-get -yq install bison flex build-essential binutils-dev libldap2-dev libcppunit-dev libicu-dev libxslt1-dev \
           zlib1g-dev libboost-regex-dev libarchive-dev libv8-dev default-jdk libapr1-dev libaprutil1-dev libiberty-dev \
-          libhiredis-dev libtbb-dev libxalan-c-dev libnuma-dev nodejs libevent-dev libatlas-base-dev libblas-dev python3-dev \
+          libhiredis-dev libtbb-dev libxalan-c-dev libnuma-dev libevent-dev libatlas-base-dev libblas-dev python3-dev \
           default-libmysqlclient-dev libsqlite3-dev libmemcached-dev libcurl4-openssl-dev pkg-config libtool autotools-dev automake \
           libssl-dev xmlstarlet
 
+        curl https://deb.nodesource.com/gpgkey/nodesource.gpg.key | sudo apt-key add -
+        #sudo apt-add-repository "deb https://deb.nodesource.com/node_16.x $(lsb_release -sc) main"
+        curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash -
```

Comment:
why are these nodejs steps different from the other 2 places where nodejs is installed below?